doc/widgets/documentembedding.md

```diff
@@ -0,0 +1,50 @@
+Document Embedding
+==================
+
+Embeds documents from input corpus into vector space by using pretrained
+[fastText](https://fasttext.cc/docs/en/crawl-vectors.html) models described in 
+E. Grave, P. Bojanowski, P. Gupta, A. Joulin, T. Mikolov,
+Learning Word Vectors for 157 Languages.
+Proceedings of the International Conference on Language Resources and Evaluation, 2018.
+
+**Inputs**
+
+- Corpus: A collection of documents.
+
+**Outputs**
+
+- Corpus: Corpus with new features appended.
+
+**Document Embedding** parses ngrams of each document in corpus, obtains embedding 
+for each ngram using pretrained model for chosen language and obtains one vector for each document by aggregating ngram embeddings using one of offered aggregators. Note that method will work on any ngrams but it will give best results if corpus is preprocessed such that ngrams are words (because model was trained to embed words).
+
+![](images/Document-Embedding-stamped.png)
+
+1. Widget parameters:
+    - Language:
+        - English: will use model trained on documents in English language
+        - Slovenian: will use model trained on documents in Slovenian language
+        - German: will use model trained on documents in German language
```

Comment:
Here I would not write a separate line for each language. I would write something like: 
```
Select the documents' language. For each language, fasttext embedder uses a separate model.
```
Or something similar.